tsconfig.json

```diff
@@ -15,7 +15,10 @@
     ],
     "baseUrl": "../",
     "noImplicitAny": false,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+
+    "esModuleInterop": false,
```

Comment:
Document in the PR that we do not change the default value for `esModuleInterop`, but rather set it explicitly (see https://www.typescriptlang.org/docs/handbook/compiler-options.html).